docs/blog/2020-11-05-zmk-sotf-3.md

```diff
@@ -0,0 +1,156 @@
+
+To learn more checkout the [Modifiers](/docs/codes/modifiers) documentation.
+
+#### Simplified Key Press Behavior
+
+In previous versions of ZMK, users needed to be careful to select between the `&kp` and `&cp` behavior in their keymaps, depending on
+if the particular keycode they wanted to send was in the "HID consumer page" or not. Forcing user to understand the difference, and get
+this right was awkward and error prone.
+
+Contributors [petejohanson] and [innovaker] have reduced this complexity, and user can now simply use `&kp` will all available codes and ZMK will properly
+handle sending the right events to the connected host.
+
+### Power Management
+
+Several important power management features have been added to ZMK, helping save power for many use cases.
+
+#### BLE Battery Level Reporting
+
+[Nicell] added the necessary [driver](https://github.com/zmkfirmware/zmk/pull/293) and [core code](https://github.com/zmkfirmware/zmk/pull/306) to send BLE battery level notifications to hosts that support displaying them. Testing seems to show this works with Windows and GNOME, but macOS does not display the battery info.
+
+#### External Power Control
+
+[megamind4089] added a new [driver](https://github.com/zmkfirmware/zmk/pull/242) and [behavior](https://github.com/zmkfirmware/zmk/pull/258) to allow user to toggle on/off the external power supplied by boards like the nrfMicro
+and nice!nano that have specialized hardware for this purpose.
+
+With this change, you can add
+
+```
+&ext_power EP_TOG
+```
+
+to toggle on/off the power to external hardware like RGB underglow or OLEDs. Check out the [external power control](/docs/behavior/power#external-power-control) docs for moore info.
+
+#### Deep Sleep
+
+[petejohanson] has contributed the initial [deep sleep](https://github.com/zmkfirmware/zmk/pull/211) support to ZMK. This work also
+included some automatic power savings by switching to PORT events on the nrf52 chips, which reduces the idle power draw, even without deep sleep. Deep sleep is currently not turned on by default, but will be soon.
+
+### Miscellaneous
+
+#### Output Selection
+
+[joelspadin] added [output selection](/docs/behavior/outputs) to allow selecting whether to send output over USB or BLE if both are connected. This should now help avoiding having "double keypresses" when your keyboard is plugged into a host.
+
+#### Bootloader Corruption Fix
+
+[Nicell] has already [blogged about this](/blog/2020/10/03/bootloader-fix), but for those that missed it, a _major_, and incredibly difficult to pin down bug involving corruption of the bootloader on devices using the Adafruit nrf52 bootloader has been fixed by [Nicell] himself. If you've encountered this bug, flashing the latest firmware should prevent it from reoccurring. Unfortunately, due to the nature of this fix, you will need to re-pair your keyboard with your hosts, as the fix involves changing where settings are stored in the flash of the controller.
```

Comment:
> [Nicell] himself

This could be misinterpreted as Nicell was the one that introduced the bug or wrote the bootloader.